src/arch/8051.md

```diff
@@ -122,9 +122,9 @@ Set a flag for a variable stored at 0x20 in internal RAM:
 Adding support for new 8051 variants
 ------------------------------------
 
-Follow these steps to add support for new 8051 variants to rizin.
+Follow these steps to add support for the new 8051 variants to rizin.
 
-1. Clone latest version of rizin
-2. In '/libr/analysis/p/analysis_8051.c' add a new entry to array 'cpu_models[]' to define a name and a memory mapping. The name of the last entry in array must be NULL
+1. Clone the latest version of rizin
+2. In '/libr/analysis/p/analysis_8051.c' add a new entry to array 'cpu_models[]' to define a name and a memory mapping. The name of the last entry in the array must be NULL
```

Comment:
Is this `libr` ok here? or should it be `librz`?

Reply:
Should be indeed `librz/`